Tweak how HTTP/3 error codes are presented for use (#123)
ietf-wg-webtrans/draft-ietf-webtrans-http3: draft-ietf-webtrans-http3.md

Since WebTransport shares the error code space with HTTP/3, WebTransport
application errors for streams are limited to an unsigned 32-bit integer,
assuming values between 0x00000000 and 0xffffffff.  WebTransport
implementations SHALL remap those error codes into the error range reserved for
WEBTRANSPORT_APPLICATION_ERROR, where 0x00000000 corresponds to 0x52e4a40fa8db,
and 0xffffffff corresponds to 0x52e5ac983162.  Note that there are code points
inside that range of form "0x1f * N + 0x21" that are reserved by {{Section 8.1

> of HTTP3}}; those have to be accounted for when mapping the error codes by

Reviewer: ```suggestion
of HTTP3}}; those have to be skipped when mapping the error codes
```

> skipping them (i.e. the two HTTP/3 error codepoints adjacent to a reserved

Reviewer: ```suggestion
(i.e. the two HTTP/3 error codepoints adjacent to a reserved
```
